src/commands/cat.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * ProtoBot -- A Discord furry bot
+ * Copyright (C) 2020  BadBoyHaloCat
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+// Imports
+import type { Client, Message } from "discord.js";
+import { MessageEmbed } from "discord.js";
+import fetch from "node-fetch";
+
+export async function run(client: Client, message: Message, args: string[], log: (mode: 'i' | 'w' | 'e', message: string) => void): Promise<void> {
+    const msg = await message.channel.send("Fetching a cat picture...")
+    fetch("https://some-random-api.ml/img/cat")
+    .then((res) => res.json())
+    .then((body) => {
+        let embed = new MessageEmbed()
+        .setTitle(`Cat for ${message.author.username}`)
+        .setImage(body.link)
+        .setTimestamp(Date.now())
+        .setColor("RANDOM")
+        msg.edit(embed)
+    })
+}
```

Comment:
```suggestion
}

```
Add newline at end of file.